.github/workflows/build-eslint-jest.yaml

```diff
@@ -21,19 +21,19 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-      - name: Node and Yarn Versions
+      - name: Print node and yarn versions
         run: |
           node --version
           yarn --version
-      - name: Install using lockfile
-        run: yarn install --frozen-lockfile
-      - name: Build
-        run: yarn build
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile --check-files
       - name: Run ESLint
         run: yarn lint
         env:
           CI: true
-      - name: Run Jest Tests
+      - name: Build
+        run: yarn build
+      - name: Run tests
```

Comment:
I'm surprised Lint is able to run prior to Build. What was the change to make this possible?

Reply:
🤦 on my side, I made it possible to run the linter without building, but for some reason I didn't bump the version over here: https://github.com/redwoodjs/redwood/commit/2341368cefefa288acfe0470070236c1bc09f74e#diff-4e9a7579f9bde879774f2c7cf3c6bcc1R7

So... it was installing an older version which made it think it was in a RedwoodJS project instead of a framework, so this happened.

Reply:
Ah! And so. many. details...